src/modules/mavlink/module.yaml

```
@@ -142,14 +150,29 @@ parameters:
             description:
                 short: Broadcast heartbeats on local network for MAVLink instance ${i}
                 long: |
-                    This allows a ground control station to automatically find the drone 
+                    This allows a ground control station to automatically find the drone
                     on the local network.
 
             type: enum
             values:
                 0: Never broadcast
                 1: Always broadcast
-                2: Only multicast        
+                2: Only multicast
             num_instances: *max_num_config_instances
             default: [1, 0, 0]
             requires_ethernet: true
+
+        MAV_${i}_FLOW_CONT:
```

Super nitpicky, but is CTRL a more common or more readable abbreviation for control? Whichever you think.

```suggestion
        MAV_${i}_FLOW_CTRL:
```